lib/tags/profile-settings.tag

```diff
@@ -435,14 +445,18 @@
                             self.refs.password.classList.remove('graphjs-error');
                             self.refs.confirmation.classList.remove('graphjs-error');
                             self.refs.password.classList.add('graphjs-success');
+                            self.refs.password.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
                             self.refs.confirmation.classList.add('graphjs-success');
+                            self.refs.confirmation.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
                             self.failMessages.includes(failMessage) && self.failMessages.splice(self.failMessages.indexOf(failMessage), 1);
                             self.successMessages.includes(successMessage) || self.successMessages.push(successMessage);
                             self.refs.submitPassword.classList.remove('graphjs-loading');
                             self.update();
                         } else {
                             self.refs.password.classList.remove('graphjs-success');
                             self.refs.confirmation.classList.remove('graphjs-success');
+                            self.refs.password.removeAttribute('data-savesuccesstext');
```

Comment:
Please remove this change. self.refs.password refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above